### ISSUE TITLE
Error with Plotly on Julia 0.7

### ISSUE THREAD

**Issue by GenaBitu** [JuliaPlots/Plots.jl#1627]
I get
```julia
julia> using Plots
julia> plot(rand(2, 6))
┌ Warning: `readstring(s::IO)` is deprecated, use `read(s, String)` instead.
│   caller = #open#298(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::typeof(readstring), ::String, ::Vararg{String,N} where N) at iostream.jl:369
└ @ Base ./iostream.jl:369
WARNING: Base.Random is deprecated, run `using Random` instead
 in module Base
┌ Warning: `Array{T}(d::NTuple{N, Int}) where {T, N}` is deprecated, use `Array{T}(undef, d)` instead.
│   caller = plot_color(::Array{RGB{FixedPointNumbers.Normed{UInt8,8}},1}) at colors.jl:24
└ @ PlotUtils ~/.julia/packages/PlotUtils/xQ9vp/src/colors.jl:24
┌ Warning: `a::Number + b::AbstractArray` is deprecated, use `a .+ b` instead.
│   caller = get_zvalues(::Int64) at color_utils.jl:104
└ @ PlotUtils ~/.julia/packages/PlotUtils/xQ9vp/src/color_utils.jl:104
┌ Warning: The start/next/done iteration protocol is deprecated. Use `iterate` instead.
│   caller = concrete_minimum(::Array{Float64,1}) at Showoff.jl:38
└ @ Showoff ~/.julia/packages/Showoff/45n6U/src/Showoff.jl:38
┌ Warning: The start/next/done iteration protocol is deprecated. Use `iterate` instead.
│   caller = concrete_maximum(::Array{Float64,1}) at Showoff.jl:60
└ @ Showoff ~/.julia/packages/Showoff/45n6U/src/Showoff.jl:60
WARNING: Base.Random is deprecated, run `using Random` instead
 in module Plots
in html_body at /home/user/.julia/packages/Plots/mqUvK/src/backends/plotly.jl
Error showing value of type Plots.Plot{Plots.PlotlyBackend}:
ERROR: UndefVarError: uuid4 not defined
Stacktrace:
 [1] getproperty at ./sysimg.jl:13 [inlined]
 [2] html_body(::Plots.Plot{Plots.PlotlyBackend}, ::Nothing) at /home/user/.julia/packages/Plots/mqUvK/src/backends/plotly.jl:909
 [3] #standalone_html#242 at /home/user/.julia/packages/Plots/mqUvK/src/backends/plotly.jl:905 [inlined]
 [4] (::getfield(Plots, Symbol("#kw##standalone_html")))(::NamedTuple{(:title,),Tuple{String}}, ::typeof(Plots.standalone_html), ::Plots.Plot{Plots.PlotlyBackend}) at ./none:0
 [5] write_temp_html(::Plots.Plot{Plots.PlotlyBackend}) at /home/user/.julia/packages/Plots/mqUvK/src/backends/web.jl:39
 [6] standalone_html_window at /home/user/.julia/packages/Plots/mqUvK/src/backends/web.jl:48 [inlined]
 [7] _display at /home/user/.julia/packages/Plots/mqUvK/src/backends/plotly.jl:936 [inlined]
 [8] display at /home/user/.julia/packages/Plots/mqUvK/src/output.jl:145 [inlined]
 [9] gui at /home/user/.julia/packages/Plots/mqUvK/src/output.jl:134 [inlined]
 [10] display(::REPL.REPLDisplay{REPL.LineEditREPL}, ::MIME{Symbol("text/plain")}, ::Plots.Plot{Plots.PlotlyBackend}) at /home/user/.julia/packages/Plots/mqUvK/src/output.jl:150
 [11] display(::REPL.REPLDisplay, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:135
 [12] display(::Plots.Plot{Plots.PlotlyBackend}) at ./multimedia.jl:287
 [13] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [14] invokelatest at ./essentials.jl:690 [inlined]
 [15] print_response(::IO, ::Any, ::Any, ::Bool, ::Bool, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:154
 [16] print_response(::REPL.AbstractREPL, ::Any, ::Any, ::Bool, ::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:139
 [17] (::getfield(REPL, Symbol("#do_respond#40")){Bool,getfield(REPL, Symbol("##50#59")){REPL.LineEditREPL,REPL.REPLHistoryProvider},REPL.LineEditREPL,REPL.LineEdit.Prompt})(::Any, ::Any, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:708
 [18] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [19] invokelatest at ./essentials.jl:690 [inlined]
 [20] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/LineEdit.jl:2261
 [21] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:1029
 [22] run_repl(::REPL.AbstractREPL, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:191
 [23] (::getfield(Base, Symbol("##831#833")){Bool,Bool,Bool,Bool})(::Module) at ./logging.jl:311
 [24] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [25] invokelatest at ./essentials.jl:690 [inlined]
 [26] macro expansion at ./logging.jl:308 [inlined]
 [27] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:340
 [28] exec_options(::Base.JLOptions) at ./client.jl:252
 [29] _start() at ./client.jl:432
```

I am running

```julia
julia> versioninfo(verbose=true)
Julia Version 0.7.0
Commit a4cb80f3ed (2018-08-08 06:46 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
      Debian GNU/Linux testing (buster)
  uname: Linux 4.17.0-1-amd64 #1 SMP Debian 4.17.8-1 (2018-07-20) x86_64 unknown
  CPU: Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz:
              speed         user         nice          sys         idle          irq
       #1  2751 MHz    1044701 s       9742 s     311215 s    3655593 s          0 s
       #2  2743 MHz    1078822 s       9882 s     311404 s    1560012 s          0 s
       #3  2743 MHz    1093800 s      10568 s     307056 s    1554178 s          0 s
       #4  2743 MHz    1039757 s      10041 s     325390 s    1588918 s          0 s
  Memory: 7.65264892578125 GB (389.78515625 MB free)
  Uptime: 81288.0 sec
  Load Avg:  5.7109375  6.2841796875  6.00439453125
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 4
  HOME = /home/user
  MATHEMATICA_HOME = /usr/local/Wolfram/Mathematica/11.0
  PATH = /home/user/.cargo/bin:/home/user/.local/bin:/home/user/awsutil:/home/user/.local/bin:/home/user/awsutil:/home/user/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/home/user/.fzf/bin
  TERM = screen-256color
  TMUX_PLUGIN_MANAGER_PATH = /home/user/.tmux/plugins/
  WINDOWPATH = 2
```

**Comment by GenaBitu** [comment]
```julia
(project) pkg> status
    Status `Project.toml`
  [587475ba] Flux v0.6.0+ [`~/.julia/dev/Flux`]
  [91a5bcdd] Plots v0.18.0
```

**Comment by daschw** [comment]
Thanks for reporting! That's already fixed on Plots master and will be released soon in Plots v0.18.1

**Comment by GenaBitu** [comment]
I tried the master branch this afternoon (CEST) and the problem was still there (that was like an hour before posting this bugreport)

Anyway, thanks for the quick response and looking forward to working with Plots in 0.7
